opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -21,7 +21,7 @@
  *
  * <p>Spans are created by the {@link Tracer#buildSpan} interface.
  */
-public interface Span extends Closeable {
```

Comment:
NB: `Span` is no longer `Closeable` to make semantics clearer around try-with-resources for `ActiveSpan` (which would become the normal/preferred way of actually using a Span).